plugin/pathvirt/pathvirt.cpp

```diff
@@ -0,0 +1,250 @@
+/* NOTE:  if you just want to insert your own code at the time of checkpoint
+ *  and restart, there are two simpler additional mechanisms:
+ *  dmtcpaware, and the MTCP special hook functions:
+ *    mtcpHookPreCheckpoint, mtcpHookPostCheckpoint, mtcpHookRestart
+ */
```

Comment:
@gc00 : ah, that comment was left from some other file that I copied as a starting point. I'll just delete the comment since it's not very relevant.